src/main/java/mikera/vectorz/impl/SparseIndexedVector.java

```diff
@@ -463,6 +539,29 @@ public void set(int i, double value) {
 			data[ip]=value;
 		}
 	}
+
```

Comment:
A sparseApplyOp method could also be used to implement other ops such as addition, multiply, etc. This should become an issue for later implementation. The only concern would be if invoking a function in a loop over non-zero elements would prevent the JVM from optimizing well. We'd maybe want function inlining?